lib/Bio/EnsEMBL/DataCheck/Checks/CheckChar.pm

```diff
@@ -0,0 +1,85 @@
+=head1 LICENSE
+
+Copyright [2018] EMBL-European Bioinformatics Institute
+
+Licensed under the Apache License, Version 2.0 (the 'License');
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an 'AS IS' BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+=cut
+
+package Bio::EnsEMBL::DataCheck::Checks::CheckChar;
+
+use warnings;
+use strict;
+
+use Moose;
+use Test::More;
+use Bio::EnsEMBL::DataCheck::Test::DataCheck;
+
+extends 'Bio::EnsEMBL::DataCheck::DbCheck';
+
+use constant {
+    NAME        => 'CheckChar',
+    DESCRIPTION => 'Check that imported names/descriptions contains only supported characters',
+};
+
+sub tests {
+    my ($self) = @_;
+
+    # Test description length
+    my $sql_length = qq/
+        SELECT *
```

Comment:
In the case of failure, all SELECTED columns will be returned as a diagnostic message, comma-separated; so "SELECT *" is fine if that is what's intended. There are only 4 columns in this table, so that's not unreasonable. I would be inclined to do "SELECT description", because then it's clear what was column was being tested.

Reply:
Well, there is effectively only 4 column, I think only description could not help finding data from report in DB. Le't keep it that way